CUDLR/Scripts/Console.cs

```diff
@@ -109,8 +144,32 @@ public class Console {
     }
 
     /* Returns the output */
-    public static string Output() {
-      return string.Join("\n", Instance.m_output.ToArray());
+    public static string Output()
+    {
+        return Output(false);
+    }
+
+    /* Returns the output, if useHtml is true will use divs instead of \n for logging */
+    public static string Output(bool useHtml) {
+        StringBuilder s = new StringBuilder();
+        foreach (ConsoleMessage m in Instance.m_output)
+        {
+            if (useHtml)
+            {
+                //Use regex to remove most valid HTML... FIX - won't catch all cases unfortunately but could be improved later
+                string htmlString = Regex.Replace(m.message, @"<[^>]*>", String.Empty);
+                s.Append("<div style='color:#" + m.color.r.ToString("X2") + m.color.g.ToString("X2") + m.color.b.ToString("X2") + ";'>");
+                s.Append(htmlString);
+                s.Append("</div>");
+            }
+            else
+            {
+                s.Append(m.message);
```

Comment:
There seems to be a slight danger of code injection (or simply parse failures) in this case. Since both HTML and non-HTML end up in the same string and it's now always interpreted as HTML rather than text in StreamingAssets/CUDLR/index.html